### ISSUE TITLE
Submission instruction 

### ISSUE THREAD

**Issue by yuanpengtu** [OpenDriveLab/OpenLane-V2#10]
Hi, when  will the submission instruction  be available?

**Comment by faikit** [comment]
Hi, it will be make avaliable before the submission period open.
We also provide an [option](https://github.com/OpenDriveLab/OpenLane-V2/blob/master/plugin/mmdet3d/baseline/datasets/openlane_v2_dataset.py#L347) in baseline model to format a pickle file for submission.

**Comment by yuanpengtu** [comment]
Hi，I wonder what hardware is used to train the baseline model， i.e., the graphics card and the used number.

**Comment by faikit** [comment]
It takes about 20 hours on 8 A100. And it could also fit into a 3090.

As asked from other platforms, to reduce memory consumption, you change the modules of the network, e.g., the DETR head to deformable DETR.
The baseline is designed for easy modification and extension. So it is recommended to get familiar with the tasks through our baseline, and then implement your awesome thoughts to solve the task.

**Comment by yuanpengtu** [comment]
It seems that I cannot reproduce the result of the baseline with 8 V100, where the batch size can only be 1 for each card. Is the batch size influence the performance? Actually I have adjusted the learning rate according to the batch size.

**Comment by yuanpengtu** [comment]
DET_l: 0.0033836588263511658, DET_t: 0.07977674901485443, TOP_ll: 0.0000, TOP_lt: 0.0000, OpenLane-V2 Score: 0.0222, F-Score for 3D Lane: 0.0818. This is the reproduced performance

**Comment by faikit** [comment]
Hi, the model is trained on some flawed data and the scores are reported on the test set.
But as mentioned in #3 that the baseline is provided for quick modification. So I think it is by no means crucial to align the scores.

Your scores are decent on this basic model based on my experience.
I would recommend you start to design your model and have fun.
